lighthouse-core/audits/dobetterweb/external-anchors-use-rel-noopener.js

```diff
@@ -47,6 +47,14 @@ class ExternalAnchorsUseRelNoopenerAudit extends Audit {
           return true;
         }
       })
+      .filter(anchor => {
+        // Ignore href's that are not real links
+        return (
```

Comment:
nit: we could probably simplify this to `!anchor.href || !anchor.href.toLowerCase().startsWith('javascript:')`